endpoint/endpoint.go

```diff
@@ -18,6 +19,7 @@ import (
 	"google.golang.org/grpc/metadata"
 	"google.golang.org/grpc/status"
 
+	zproto "github.com/zenoss/zenoss-protobufs/go/cloud/data-registry"
```

Comment:
Looks like there is a problem with the - in data-registry if I try to use it without an alias . That said, I can choose the alias to be data_registry instead of zproto

Reply:
Yes, using data_regisrty as an alias would be fine